spec/site/tlc/alarm_spec.rb

```diff
@@ -74,6 +74,40 @@
         end
       end
     end
+    
+    it 'can be suspended', :script, sxl: '>=1.0.7' do |example|
+      
+      alarm_code_id = 'A0301'   # what alarm to expect
+      component = 'KK+AG9998=001DL003'
```

Comment:
component ids should be read from the config, not hardcoded into tests. they differ depending on the equipment you test. the detector logic component ids are already in the config, but in case you need a specific DL for this test, you can the detector logic nr as a specific config element.